app/controllers/collection_numbers_controller.rb

```diff
@@ -262,12 +276,12 @@ def normalize_parameters
     end
   end
 
-  def name_and_number_free?
+  def name_and_number_used?
     @other_number = CollectionNumber.where(
       name: @collection_number.name,
       number: @collection_number.number
     ).first
-    !@other_number || @other_number == @collection_number
+    @other_number && @other_number != @collection_number
```

Comment:
Not a blocker, but I prefer explicit parens over relying on precedence rules when it's not really, really obvious.  In this case I believe that means `@other_number && (@other_number != @collection_number)`

Reply:
[Commit #5be922f](https://github.com/MushroomObserver/mushroom-observer/pull/571/commits/5be922f) hopefully finesses/avoids this with:
```ruby
@other_number&. != @collection_number
``` 
I also wonder if
```ruby
@other_number && (@other_number != @collection_number)
```
would be a RuboCop offense, which would mean that we'd to disable and re-enable the cop in order to avoid a complaint and autocorrection.